charts/kafka/Chart.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Crown Copyright
```

Comment:
Given line 15 then this is not covered under Crown Copyright as tax payers money has not written this code